### PR TITLE
Syslog

### PR DESCRIPTION
**Pull request by jforissier**

### CODE REVIEW

File: tee-supplicant/src/tee_supplicant.c [hunk 1]
```diff
@@ -410,6 +431,9 @@ int main(int argc, char *argv[])
 			case RPC_CMD_SHM_FREE:
 				process_free(&request);
 				break;
+			case RPC_CMD_LOG:
+				process_log(&request);
+				break;
 			default:
 				EMSG("Cmd [0x%" PRIx32 "] not supported",
```
Comment (vchong) [inline tee-supplicant/src/tee_supplicant.c]: This is not part of the PR but is missing a format specifier.

Comment (jforissier) [inline tee-supplicant/src/tee_supplicant.c]: `PRIx32` *is* the format specifier ;)
```
$ git grep -n "define PRIx32"
lib/libutils/isoc/include/inttypes.h:66:#define PRIx32          "x"
```

Comment (vchong) [inline tee-supplicant/src/tee_supplicant.c]: Oops.. ok. Sorry for the noise.

**Comment by jforissier** [comment]
Closing since this is not the correct approach (see comments in https://github.com/OP-TEE/optee_os/issues/810).
